package.json

```diff
@@ -621,6 +621,11 @@
           "default": true,
           "description": "Align assignment statements in a hashtable or a DSC Configuration."
         },
+        "powershell.codeFormatting.useCorrectCasing": {
+          "type": "boolean",
+          "default": true,
+          "description": "Correct casing of cmdlets."
```

Comment:
Minor English "nit" … "Correct" in this case sounds like it might be a verb instead of an adjective. I suggest "Use correct casing for cmdlets." - or something like that.

Reply:
Fair point. Done